package.json

```diff
@@ -68,6 +69,7 @@
     "build:css": "lerna exec --ignore design-system --stream -- 'postcss src/*.pcss --dir dist/ --ext css'",
     "build:js": "lerna exec --ignore design-system --stream -- 'babel src/ --out-dir dist/ --source-maps --config-file ../../babel.config.js'",
     "heroku-postbuild": "npm run --prefix design-system build",
+    "heroku-cleanup": "npm prune",
```

Comment:
I have a feeling that npm does this automatically after install, and it's actually `npm prune --production` that we need to remove development dependencies?  I think!